static/css/media-queries.css

```diff
@@ -0,0 +1,28 @@
+/*  We can't use standard breakpoints as the left panel navigation in Jira hijacks some of the screen */
+@media (max-width: 1530px) {
+    .jiraConfiguration__table__cell__settings__ellipsis {
+      right: 0;
+    }
+  }
+
+  @media (max-width: 800px) {
+    .jiraConfiguration {
+      padding: 2em;
+    }
+
+    .jiraConfiguration__table__cell__settings__ellipsis {
+      right: 0.2em !important;
+    }
+  }
+
+  @media (max-width: 540px) {
+    .getConfiguration__tableHeader {
```

Comment:
There is a duplicate of this styling in global-OLD.css due to adding the FF after this work was done. Easier to just leave the hbs files pointing at the resources they need than to move styles around.